go.mod

```diff
@@ -3,22 +3,17 @@ module github.com/jodevsa/wireguard-operator
 go 1.16
```

Comment:
Do you need go.sum and go.mod to be updated? if not, I would prefer to postpone updating any dependencies in a separate PR if possible.